src/cmake/compiler.cmake

```diff
@@ -469,7 +469,7 @@ if (CLANG_FORMAT_EXE)
     # message (STATUS "clang-format file list: ${FILES_TO_FORMAT}")
     file (COPY ${CMAKE_CURRENT_SOURCE_DIR}/.clang-format
           DESTINATION ${CMAKE_CURRENT_BINARY_DIR})
-    add_custom_target (clang-format
+    add_custom_target (osl-clang-format
```

Comment:
If you don't mind, let's call this "run-clang-format".

I tend to copy and/or synchronize the more generic cmake scripts between multiple projects, so it's very handy to not have too many project-specific names littering the code like this that could in theory be applied to any project.

Reply:
I have zero preference in this so `run-clang-format` is fine